core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -795,6 +796,7 @@
         "alluxio.user.ufs.block.read.concurrency.max";
     public static final String USER_UFS_BLOCK_OPEN_TIMEOUT_MS =
         "alluxio.user.ufs.block.open.timeout.ms";
+    public static final String USER_SHORT_CIRCUIT_ENABLED = "alluxio.user.short.circuit.disabled";
```

Comment:
Is this property for testing only? If so, would it be possible to implement this some other way?